plugins/embed-optimizer/class-embed-optimizer-tag-visitor.php

```diff
@@ -283,4 +195,134 @@ private function reduce_layout_shifts( OD_Tag_Visitor_Context $context ): void {
 			$processor->append_head_html( sprintf( "<style>\n%s\n</style>\n", join( "\n", $style_rules ) ) );
 		}
 	}
+
+	/**
+	 * Gets preconnect URLs based on embed type
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param OD_HTML_Tag_Processor $processor Processor.
+	 * @return array<non-empty-string> Array of URLs to preconnect to.
+	 */
+	private function get_preconnect_urls( OD_HTML_Tag_Processor $processor ): array {
+		/*
+		* The following embeds have been chosen for optimization due to their relative popularity among all embed types.
```

Comment:
It seems the comment lines here are missing an ` ` space before the `*`

Reply:
Oops. Fixed it in 0b4150c.